test/src/unit-capi-config.cc

```diff
@@ -238,6 +238,7 @@ void check_save_to_file() {
   ss << "sm.enable_signal_handlers true\n";
   ss << "sm.io_concurrency_level " << std::thread::hardware_concurrency()
      << "\n";
+  ss << "sm.max_tile_overlap_size 314572800\n";
```

Comment:
Do we want this to be 500MB?

Reply:
I dropped it to 300MB because this isn't a strict limit. If each range generates an identically sized tile overlap, we might allocate 600MB since the size doubles in each loop.